app/jobs/create_cap_proposal_job.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCapProposalJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(auction_id)
+    auction = Auction.find(auction_id)
+    auction_presenter = AuctionPresenter.new(auction)
+    cap_proposal_url = CreateCapProposal.new(auction_presenter).perform
+    auction.update(cap_proposal_url: cap_proposal_url)
```

Comment:
Could we have this be updated as part of the `perform` method? Could also define the presenter within it